### ISSUE TITLE
How to get the last message in ContactsView?

### ISSUE THREAD

**Issue by hoangdv1995** [tinode/react-js#10]
I want to show the last message on Contacts View.
Help me, thanks!

**Comment by hoangdv1995** [comment]
Sorry, but i want to get the last message of contact in Tinode.getMeTopic().contacts(????);

**Comment by or-else** [comment]
If you want to show the text of the last message in contact list then you have to do it yourself. When you receive a notification of a new message, fetch it and show it in the contact list. You can do it purely on the client or you can add it to `private.comment` making it available to all user's client devices.

The feature cannot be done server-side because the messages can be end-to-end encrypted. In such a case the server will have no access to messages at all.

**Comment by hoangdv1995** [comment]
can't get public name of topic?
let topic = Tinode.getTopic(topicName);
console.log(topic.public.fn);      => null

**Comment by or-else** [comment]
Have you subscribed to the topic first?

**Comment by Coolister-Ye** [comment]
I'm facing the same question. But from the comments above, i still don't know how to do it. I checked the tinode web project code. It's show that I can get each topic's latest message from "this.tinode.getMeTopic().contacts((c) => console.log(c.latestMessgae(true)))" and I have subscribed to topic me. However i can only get undefined from here. Can anyone show me a way to get the latest message from each topic? Thanks.
